api/src/main/java/org/apache/cloudstack/api/command/user/vm/CreateVMScheduleCmd.java

```diff
@@ -55,7 +54,7 @@ public class CreateVMScheduleCmd extends BaseCmd {
     @Parameter(name = ApiConstants.SCHEDULE,
             type = CommandType.STRING,
             required = true,
-            description = "Schedule for action on VM in cron format")
+            description = "Schedule for action on VM in cron format. e.g. '0 15 10 * *'")
```

Comment:
```suggestion
            description = "Schedule for action on VM in cron format. e.g. '0 15 10 * *' for 'at 15:00 on 10th day of every month'")
```
@vishesh92 would it make more sense?